ha/const.py

```diff
@@ -293,3 +296,8 @@ class AlertEventConstants(Enum):
     ALERT_RESOURCE_TYPE = f"alert{_DELIM}resource_type"
     IEM_COMPONENTS = f"iem{_DELIM}components"
     IEM_MODULES = f"iem{_DELIM}modules"
+
+class BMC_CREDENTIALS(Enum):
```

Comment:
move this to ipmi fencing agent only

for ha it is abstract detail
all files in utils should be independent of ha

Reply:
Moved to ipmi_fencing_agent class.